app/pages/Accounts/AccountDetailsPage/StakingDetails/StakingDetails.tsx

```diff
@@ -0,0 +1,226 @@
+            <span className="body2">{value}</span>
+        </div>
+    );
+}
+
+interface ValuesProps<T> {
+    details: T;
+}
+
+const formatCommission = (value: number) =>
+    `${toFixed(3)((value * 100).toString())}%`;
+
+interface BakerValuesProps extends ValuesProps<AccountBakerDetails> {
+    isDelegationProtocol: boolean;
+}
+
+function BakerValues({ details, isDelegationProtocol }: BakerValuesProps) {
+    return (
+        <>
+            <Value
+                title="Baker stake"
+                value={displayAsCcd(details.stakedAmount)}
+            />
+            <Value title="Baker ID" value={details.bakerId.toString()} />
+            <Value
+                title="Rewards wil be"
+                value={
+                    details.restakeEarnings
+                        ? 'Added to stake'
+                        : 'Added to public balance'
+                }
+            />
+            {isDelegationProtocol && (
+                <>
+                    <Value
+                        title="Delegation pool status"
+                        value={displayPoolOpen(
+                            (details as AccountBakerDetailsV1).bakerPoolInfo
+                                .openStatus
+                        )}
+                    />
+                    <Value
+                        title="Transaction fee commission"
+                        value={formatCommission(
+                            (details as AccountBakerDetailsV1).bakerPoolInfo
+                                .commissionRates.transactionCommission
+                        )}
+                    />
+                    <Value
+                        title="Baking reward commission"
+                        value={formatCommission(
+                            (details as AccountBakerDetailsV1).bakerPoolInfo
+                                .commissionRates.bakingCommission
+                        )}
+                    />
+                    <Value
+                        title="Finalization reward commission"
+                        value={formatCommission(
+                            (details as AccountBakerDetailsV1).bakerPoolInfo
+                                .commissionRates.finalizationCommission
+                        )}
+                    />
+                </>
+            )}
+        </>
+    );
+}
+
+function DelegatorValues({ details }: ValuesProps<AccountDelegationDetails>) {
+    return (
+        <>
+            <Value
+                title="Delegation amount"
+                value={displayAsCcd(details.stakedAmount)}
+            />
+            <Value
+                title="Target pool"
+                value={displayDelegationTarget(details.delegationTarget)}
+            />
+            <Value
+                title="Rewards wil be"
+                value={
+                    details.restakeEarnings
+                        ? 'Added to delegation amount'
+                        : 'Added to public balance'
```

Comment:
lets discuss these :upside_down_face:

Reply:
Please do... change these and the baking ones to use same terminology as the transaction flows